_data/channels.csv

```diff
@@ -72,15 +65,12 @@ Conscious Good,Active,2018-01-16,North America,"AVOD,SVOD,TVOD",Wholesome,consci
 FlixFling,Active,2018-06-05,North America,"SVOD,TVOD",,flixfling.svg
 Telus,Active,2017-11-02,North America,SVOD,Educational,telus.svg
 Realeyz,Active,2015-06-28,Europe,TVOD,,
-Amazon UK,Active,2016-05-01,Europe,"SVOD,TVOD","Arthouse,Culture,Docs,Educational,Horror,LGBT,Shorts,Wholesome",
-Amazon RoW Portuguese,Active,2018-04-18,Europe,"SVOD,TVOD","Arthouse,Culture,Docs,Educational,Horror,LGBT,Shorts,Wholesome",
 The Love Destination,Active,2018-09-17,Oceania,SVOD,,
 TikiLive,Active,2018-03-14,North America,SVOD,,
-Amazon RoW Italian,Active,2018-04-18,Europe,"SVOD,TVOD","Arthouse,Culture,Docs,Educational,Horror,LGBT,Shorts,Wholesome",
 PlayNow Media,Active,2018-09-12,North America,"AVOD,SVOD",Horror,
 InsideOutside,Active,2018-01-18,North America,"AVOD,SVOD",Cultural,
-Amazon RoW English,Active,2017-04-01,Europe,"SVOD,TVOD","Arthouse,Culture,Docs,Educational,Horror,LGBT,Shorts,Wholesome",
 Bidslate,Active,2018-04-02,North America,"SVOD,TVOD","Arthouse,Culture,Docs,Educational,Horror,LGBT,Shorts,Wholesome",
 FreebieTV,Active,2018-10-31,Europe,AVOD,,freebie-tv.png
 Screambox,Active,2017-08-30,North America,"AVOD,SVOD",Horror,
 American Classics,Active,2018-02-21,Europe,AVOD,,
+VIDI SPACE,Active,2020-4-22,North America,"SVOD,TVOD",,

```

Comment:
Without logo the site might break. Please add the logo @beaujeanfilmhub

Reply:
This has been corrected.